02.calendar/cal.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+
+function parseInput(argv) {
+  const date = new Date();
+  const { m: month = date.getMonth() + 1, y: year = date.getFullYear() } =
+    minimist(argv.slice(2));
+  if (isNaN(year) || year < 1 || year > 9999) {
+    console.log(`cal: year '${year}' not in range 1..9999`);
+    return { year: null, month: null };
+  }
+  if (isNaN(month) || month < 1 || month > 12) {
+    console.log(`cal: ${month} is neither a month number (1..12) nor a name`);
+    return { year: null, month: null };
+  }
+  return { year: year, month: month - 1 };
```

Comment:
プロパティの名前と、値の変数が同じ場合は省略記法を使ってください 🙏 

```suggestion
  return { year, month: month - 1 };
```